core/src/repair_service.rs

```diff
@@ -479,14 +479,20 @@ impl RepairService {
                     .collect();
                 info!("repair_stats: {:?}", slot_to_count);
                 if repair_total > 0 {
+                    let nonzero_num = |x| if x == 0 { None } else { Some(x) };
                     datapoint_info!(
                         "repair_service-my_requests",
-                        ("repair-total", repair_total, i64),
                         ("shred-count", repair_stats.shred.count, i64),
                         ("highest-shred-count", repair_stats.highest_shred.count, i64),
                         ("orphan-count", repair_stats.orphan.count, i64),
-                        ("repair-highest-slot", repair_stats.highest_shred.max, i64),
-                        ("repair-orphan", repair_stats.orphan.max, i64),
+                        ("shred-slot-max", nonzero_num(repair_stats.shred.max), Option<i64>),
+                        ("shred-slot-min", nonzero_num(repair_stats.shred.min), Option<i64>),
```

Comment:
Looks like `RepairStatsGroup.min` is always zero.

Reply:
Yep, looks that way. Updated.